src/scripts/components/selected-layer-list-item/selected-layer-list-item.styl

```diff
@@ -21,6 +21,7 @@
   border-radius: 50%
 
 .removeIcon
+  display: contents
```

Comment:
What do you think of this?

Without it, the remove button ist not centered in the div, even though all items are center aligned: 
![image](https://user-images.githubusercontent.com/45818654/77407180-08f57180-6db6-11ea-8890-bd208e8462f5.png)

with display: contents:
![image](https://user-images.githubusercontent.com/45818654/77407275-2de9e480-6db6-11ea-926b-c8df6e04e9af.png)

Reply:
hmm support for this seems to be ok but not really good. What's the problem with aligning it with display: flex or something similiar? I would prefer the "normal" way

Reply:
Okay 👍